real_time/surface_src/java_src/BuggyRos/src/test/java/NumberSource.java

```diff
@@ -0,0 +1,73 @@
+
+import com.roboclub.robobuggy.ros.Node;
+import com.roboclub.robobuggy.ros.Publisher;
+
+public class NumberSource implements Node {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2319545999/issues/source?bid=2938665&fileBranchId=3199623#l5)